### PR TITLE
upgrade all print statements to python3

### PR DESCRIPTION
**Pull request by loftusa**
small hotfix to get print statements python3 compatible.
Did this with a `ctrl+f` across the whole package, so this should take care of all the print statements in ndmg

### CODE REVIEW

File: ndmg/register/gen_reg.py [hunk 1]
```diff
@@ -861,9 +861,9 @@ def dwi2atlas(self, clean=False):
         nib.save(self.b0_out, self.b0)
 
         # Applies skull stripping to T1 volume, then EPI alignment to T1
-        print("calling mgru.extract_brain on {}, {}").format(
+        print("calling mgru.extract_brain on {}, {}".format(
             self.t1w, self.t1w_brain
-        )  # t1w = in, t1w_brain = out
+        ))  # t1w = in, t1w_brain = out
```
Comment (loftusa) [inline ndmg/register/gen_reg.py:866]: good

File: ndmg/register/gen_reg.py [hunk 2]
```diff
@@ -872,17 +872,17 @@ def dwi2atlas(self, clean=False):
         mgru.align_epi(self.dwi, self.t1w, self.t1w_brain, self.temp_aligned)
 
         # Applies linear registration from T1 to template
-        print("calling mgru.align on {}, {}, {}").format(self.t1w, self.atlas, self.xfm)
+        print("calling mgru.align on {}, {}, {}".format(self.t1w, self.atlas, self.xfm))
```
Comment (loftusa) [inline ndmg/register/gen_reg.py:875]: good

File: ndmg/register/gen_reg.py [hunk 3]
```diff
@@ -872,17 +872,17 @@ def dwi2atlas(self, clean=False):
...
         mgru.align(self.t1w, self.atlas, self.xfm)
 
         # Applies combined transform to dwi image volume
-        print("calling mgru.applyxfm on {}, {}, {}, {}").format(
+        print("calling mgru.applyxfm on {}, {}, {}, {}".format(
```
Comment (loftusa) [inline ndmg/register/gen_reg.py:879]: good

File: ndmg/register/gen_reg.py [hunk 4]
```diff
@@ -872,17 +872,17 @@ def dwi2atlas(self, clean=False):
...
             self.atlas, self.temp_aligned, self.xfm, self.temp_aligned2
-        )
+        ))
```
Comment (loftusa) [inline ndmg/register/gen_reg.py:881]: good

**Review by dPys — APPROVED** [review]

File: ndmg/register/gen_reg.py [hunk 5]
```diff
@@ -872,17 +872,17 @@ def dwi2atlas(self, clean=False):
...
         mgru.applyxfm(self.atlas, self.temp_aligned, self.xfm, self.temp_aligned2)
-        print("calling mgru.resample on {}, {}, {}").format(
+        print("calling mgru.resample on {}, {}, {}".format(
```
Comment (loftusa) [inline ndmg/register/gen_reg.py:883]: good

File: ndmg/scripts/ndmg_dwi_pipeline.py [hunk 6]
```diff
@@ -419,16 +419,16 @@ def ndmg_dwi_worker(
         # Align DWI volumes to Atlas
         print("Aligning volumes...")
         reg = mgr.dmri_reg_old(dwi_prep, gtab, t1w, atlas, aligned_dwi, namer, clean)
-        print("Registering DWI image at {} to atlas; aligned dwi at {}...").format(
+        print("Registering DWI image at {} to atlas; aligned dwi at {}...".format(
```
Comment (loftusa) [inline ndmg/scripts/ndmg_dwi_pipeline.py:422]: good

File: ndmg/scripts/ndmg_dwi_pipeline.py [hunk 7]
```diff
@@ -419,16 +419,16 @@ def ndmg_dwi_worker(
...
+        ))  # alex  # TODO: make sure dwi_prep is what is being registered
         reg.dwi2atlas()
 
         # -------- Tensor Fitting and Fiber Tractography ---------------- #
         print("Beginning tractography...")
         # Compute tensors and track fiber streamlines
-        print("aligned_dwi: {}").format(aligned_dwi)
-        print("gtab: {}").format(gtab)
+        print("aligned_dwi: {}".format(aligned_dwi))
+        print("gtab: {}".format(gtab))
```
Comment (loftusa) [inline ndmg/scripts/ndmg_dwi_pipeline.py:431]: both good

File: ndmg/scripts/ndmg_dwi_pipeline.py [hunk 8]
```diff
@@ -443,8 +443,8 @@ def ndmg_dwi_worker(
 
         # Save streamlines to disk
         print("Saving streamlines: " + streams)
-        print("streamlines: {}").format(streamlines)
-        print("streams: {}").format(streams)
+        print("streamlines: {}".format(streamlines))
+        print("streams: {}".format(streams))
```
Comment (loftusa) [inline ndmg/scripts/ndmg_dwi_pipeline.py:447]: both good

File: ndmg/scripts/ndmg_dwi_pipeline.py [hunk 9]
```diff
@@ -454,7 +454,7 @@ def ndmg_dwi_worker(
         streamlines = Streamlines(
             streamlines
         )  # alex  # to try to make the streamlines variable be the same thing as the native space one
-        print("atlas location: {}").format(atlas)
+        print("atlas location: {}".format(atlas))
```
Comment (loftusa) [inline ndmg/scripts/ndmg_dwi_pipeline.py:457]: good

**Pull request merged**